.gitignore

```diff
@@ -32,8 +32,10 @@ data_gen/OpenLabeling/main/current_zip.txt
 venv/
 
 # Train Gen
-
 train_gen/training_data/data/
+train_gen/obj.data
+train_gen/yolov3.cfg
+train_gen/weights_zips/darknet*/
```

Comment:
Just want to double check you meant to have the wildcard (*) after darknet and not after the /

Reply:
Yeah, want to keep the full folder to keep the structure but ignore the output of the script